### ISSUE TITLE
Accessibility no longer works on main tree

### ISSUE THREAD

**Issue by justanotheranonymoususer** [gitextensions/gitextensions#9852]
### Environment

- Git Extensions 3.5.4.12724
- Build 65f01f39982ccb121c4df7d4b00b506939553958
- Git 2.31.1.windows.1
- Microsoft Windows NT 10.0.19044.0
- .NET Framework 4.8.4420.0
- DPI 96dpi (no scaling)

### Issue description

In previous versions, I could use my screen reader. Now it only says "Row 123" instead of the commit message. See screenshot with inspect.exe.

![image](https://user-images.githubusercontent.com/5781692/152169002-4f3c3eb4-8755-464a-8be7-c8bf290f3712.png)

### Steps to reproduce

Run inspect.exe (C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64\inspect.exe) and hover over a commit row. Commit message is not accessible.

### Did this work in previous version of GitExtensions?

Yes, but I don't remember which unfortunately.

**Comment by RussKie** [comment]
To be honest I'm surprised it ever worked, as the accessibility isn't on the front mind of any contributor unless they have vested interests in it. And since the team consists of volunteers, expecting anyone to dedicate efforts to accessibility work is akin herding cats - i.e. it's largely a futile exercise. I know if sucks for users with accessibility needs, but it's a hard cold reality...
The project is in need of an accessibility champion.

**Comment by justanotheranonymoususer** [comment]
I understand, but the list of commits is the main control of the
program, so maybe it's not difficult to tweak at least only this
control to support accessibility. In any case, thanks for the reply
and for this great project.

**Comment by mstv** [comment]
I have checked back down to v2.48: the `Name` always is "Message Row xxx".
Changed in one of the WinForms or .NET updates?
